optimum/neuron/utils/input_generators.py

```diff
@@ -18,7 +18,7 @@
 
 import torch
 
-from ...utils import (
```

Comment:
for the optimum subpkg design, previous path was preferred.

Reply:
No, in the contrary: this is precisely the kind of relative import that is the root cause of all the issues we have.
optimum is a namespace, not a package, and we should never use relative imports between packages inside the namespace.

Reply:
Could you elaborate on what issues? optimum namespace pkg is mandatory for all subpkg, tbh absolute or relative import both are ok for me, and absolute paths are what I used beginning this subpkg, but it's the rule of thumb that I was told that I adopted relative import, if it's causing any issue, I will change those imports to absolute.

Reply:
Yes, you must use absolute, because otherwise you can't install the package in editable mode (among other side-effects).